bigbluebutton-html5/imports/ui/components/actions-bar/create-breakout-room/component.jsx

```diff
@@ -871,6 +871,7 @@ class BreakoutRoom extends PureComponent {
     return this.getUserByRoom(room)
       .map((user) => (
         <p
+          tabIndex={0}
```

Comment:
Setting the `tabIndex`'s to `0` for the names in the list will force a keyboard user into tabbing through each name in a list before being able to move to the next. This can be problematic with  a lot of users in the list. We should set this `tabIndex` to `-1`.